includes/ApplHandlingByID.php

```diff
@@ -299,6 +299,163 @@ function accept ( $id, $type, $user )
 }
 
 
+function forwardApprover ( $id, $type, $user )
+{
+	require "../../LocalSettings.php";
+	require "../Globals.php";
+
+	$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
+
+	if ( $sqlconn->connect_errno )
+	{
+		echo "Internal Server Error, Sorry for inconvenience.";
+		die();
```

Comment:
__[phpmd]__
* The function forwardApprover() contains an exit expression.

<!-- Comment by [SideCI](https://sideci.com) -->